lib/out.js

```diff
@@ -39,5 +44,6 @@ module.exports = {
     writeln: writeln,
     write: write,
     ln: ln,
-    padding: padding
+    padding: padding,
+    getStream: getStream
```

Comment:
I must say though, this is a pretty long winded way of moving `process.stdout` from lib/constants.js to lib/observatory.js :(